arviz/tests/test_plots.py

```diff
@@ -606,6 +606,13 @@ def test_plot_violin_discrete(discrete_model):
     assert axes.shape
 
 
+def test_plot_autocorr_short_chain():
+    """Check that """

```

Comment:
nitpicking. This looks incomplete, maybe we can just remove it...

Reply:
Good nitpick, ti wasn't complete. I completed the sentence. Thanks for the catch